scripts/TestClient.py

```diff
@@ -4,7 +4,6 @@
 
 from tuxemon.mod_manager import Manager
 import os
-import readline
```

Comment:
Note that `readline` is imported for the input history (accessible by the up/down arrows)